GREEDY/View/MainScreen.cs

```diff
@@ -1,87 +1,50 @@
-﻿using GREEDY.Controllers;
-using System;
-using System.Collections.Generic;
+﻿using System;
 using System.Windows.Forms;
-using System.Configuration;
-using System.Data;
-using GREEDY.Models;
+using GREEDY.Services;
+using GREEDY.DataManagers;
 
-namespace GREEDY
+namespace GREEDY.View
 {
-    public partial class Greedy : Form
+    public partial class MainScreen : Form
     {
-        public Greedy()
+        private readonly IReceiptService _receiptService;
+        private readonly IImageGetter _photoImageGetter;
+        private readonly IImageGetter _fileImageGetter;
+
+        public MainScreen(ReceiptService receiptService)
         {
+            _receiptService = receiptService;
+            _photoImageGetter = new PhotoImageGetter();
+            _fileImageGetter = new FileImageGetter();
             InitializeComponent();
         }
 
-        private void MainScreen_Load(object sender, EventArgs e)
+        private void InserFile_Button_Click(object sender, EventArgs e)
```

Comment:
Application crashes if user clicks cancel in OpenFileDialog

Reply:
Yes, because If we cancel i sent a null as bitmat file. Need to implement this in the future